README.md

```diff
@@ -18,3 +18,66 @@ Open [http://localhost:3000](http://localhost:3000) with your browser to see the
 You can start editing the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
 
 This project uses [`next/font`](https://nextjs.org/docs/app/building-your-application/optimizing/fonts) to automatically optimize and load [Geist](https://vercel.com/font), a new font family for Vercel.
+
+## Structure and Design
+
+### Components
+`Side-menu Bar`
+- To navigate quickly through the site
+`Footer`
+- For web information
+
+
+### Pages
+`Home`
+- Contains brief introduction description
+- Buttons to go to main church pages
+
+`About`
+- Location
+- Service schedules
+- Church leaders
+
+`Events`
+- Information about upcoming events
+- General events
+- Group events
+- Users can create group events
+  - `name`
+  - `description`
+  - `group`
+  - `startDate`
+  - `endDate`
+- Admins can create general events
+
+`Connect`
+- Group information (ex. Sunday school, Hi-C, CU, Adults)
+- Group leader information
+  - Users can modify the leader information for the group
+  - Admins can modify all groups
+
+`Contact`
+- Contact information
+- Leaders to speak to or contact
+
+
+### Users
+`General User`
+- Access to modifying their respective group pages/descriptions (ex. CU user can modify CU info)
```

Comment:
Too general, can be ruined by more trolls. 

I suggest either:
- approval of user login
- a history of changes

Reply:
This will be solved for now in my previous comment.